deep-learning/README.md

```diff
@@ -18,6 +18,16 @@ In these courses, you will learn the foundations of Deep Learning, understand ho
 1. [Introduction to Computer Vision](https://www.udacity.com/course/introduction-to-computer-vision--ud810)
 2. [CS231n: Convolutional Neural Networks for Visual Recognition](http://cs231n.stanford.edu/)
 
+### Roadmap to Deep Learning with Images
```

Comment:
You can say Deep Learning for Visual Recognition. Perception is a bigger category. Prerequisites will change accordingly.